build.gradle

```diff
@@ -2,4 +2,5 @@
 plugins {
     id 'com.android.application' version '7.4.1' apply false
     id 'com.android.library' version '7.4.1' apply false
+    id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin' version '2.0.1' apply false
```

Comment:
```suggestion
```

I don't believe you used that anywhere. Indeed the `./local.properties` file is already generated by the `.cirrus.yml` CI. Indeed this plugin is another way to expose the API key using a compiled binary. Anyway API keys are still recoverable by decompiling an APK so well...

Reply:
The same as above